letta/server/server.py

```diff
@@ -181,13 +184,10 @@ def run_command(self, user_id: str, agent_id: str, command: str) -> Union[str, N
     # TODO: don't rely on config storage
     engine = create_engine("sqlite:///" + os.path.join(config.recall_storage_path, "sqlite.db"))
 
```

Comment:
only sqllite will have databases automatically created, if you are using postgres you need to run alembic migrate